modules/install-roxctl-cli-linux.adoc

```diff
@@ -7,21 +7,27 @@
 
 You can install the `roxctl` CLI binary on Linux by using the following procedure.
 
+[NOTE]
+====
+`roxctl` CLI for Linux is available for `x86_64`, `amd64`, `ppc64le`, and `s390x` architectures.
+====
+
 .Procedure
 
-. Download the `roxctl` CLI:
+. Query the architecture where to install `roxctl` and set `$arch` environment variable:
 +
 [source,terminal,subs=attributes+]
 ----
 $ arch="$(uname -m | sed "s/x86_64//")"; arch="${arch:+-$arch}"
 ----
+
+. Download the `roxctl` CLI:
 +
 [source,terminal,subs=attributes+]
 ----
 $ curl -f -o roxctl "https://mirror.openshift.com/pub/rhacs/assets/{rhacs-version}/bin/Linux/roxctl${arch}"
 ----
-+
-NOTE: `roxctl` CLI for Linux is available for `amd64`, `ppcl64le`, and `s390x` architectures.
+
```

Comment:
@msugakov I moved the note ahead of the procedure. Alternatively (or in addition), we could also add a callout.

```suggestion
<1> Valid values for {arch} are `x86_64`, `amd64`, `ppc64le`, and `s390x`.
```
and add <1> to line 28

Reply:
Re moving the note: fine with me.

Re adding the callout: it is not necessary, because the expression we provide
```bash
$ arch="$(uname -m | sed "s/x86_64//")"; arch="${arch:+-$arch}"
```
must set `$arch` for the user correctly, the user does not need to look or alter `$arch`.